app/src/components/NavBar.tsx

```diff
@@ -2,9 +2,12 @@ import React from 'react';
 
 import { Alignment, Box, Direction, Image, PaddingSize, Spacing, Stack } from '@kibalabs/ui-react';
 
+import { useAccountIds } from '../accountContext';
 import { Account } from './Account';
 
 export const NavBar = (): React.ReactElement => {
+  const accountIds = useAccountIds();
```

Comment:
ah so in mdtp i implemented it so you can link multiple accounts. but actually this never works. can you add another function in AccountContext called `useAccountId` which returns the first in the array if it exists